### ISSUE TITLE
Syntax highlight breaks upon switching retype tabs

### ISSUE THREAD

**Issue by AdamBrianBright** [retypeapp/retype#119]
**Retype version used**: `1.8.0`, `1.8.2`

**The problem**

Code blocks and snippets don't highlight inside tabs

Since my docs are under gitlab auth, here's a **public example** that I've found [in discussions](https://github.com/retypeapp/retype/discussions/106): https://bashly.dannyb.co/usage/writing-your-scripts/

**Screenshots**

| Expected behaviour                                                                                              | Actual behaviour                                                                                                |
| --------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------- |
| ![image](https://user-images.githubusercontent.com/15821800/128893762-d14aaec3-a61d-4c40-a2ba-45ee0de7e6ce.png) | ![image](https://user-images.githubusercontent.com/15821800/128893810-9cc2d6fc-b88e-4d9a-a36a-e1dfe7998c95.png) |

**Steps to reproduce**

1. Create a page with code inside second+ _tab_ like this:
```markdown
    +++ Input
    
    ```json5
    {
     "username": "Adam_Bright",
    }
    ```
    
    +++ Output
    
    ```json5
    {
     "uuid": "12345678-1234-5678-1234-567812345678",
    }
    ```
    
    +++
```
2. Build and open docs on random page
3. Go to your page with _code_ in _tabs_ via _navigation_, _references_, etc., **do not open it directly as new window/tab**
4. Switch _tab_ to **Output**
5. Your code is not highlighted
6. Reload page and switch _tab_ again
7. Your code is highlighted

**Expected behaviour**

1. Open random page
2. Navigate to a page with code in _tabs_
3. Switch _tab_ to second+
4. Your code is highlighted

**Comment by geoffreymcgill** [comment]
Hi @AdamBrianBright. Thanks for the report. 

We have noticed this issue as well, although we have not had the opportunity to fully investigate. We do think it's only an issue during development and likely has something to do with the live reload functionality. 

During production with a live published website, I don't think issue can happen, but that is yet to be confirmed as well.

Once we figure out why this happening under the local dev server (`retype watch`), I will post an update. With some luck, it will be an easy fix and we can include in the fix in the next Retype release.

**Comment by AdamBrianBright** [comment]
I can't figure why it happens, but it only occurs when you navigate to another page, and it doesn't matter whether it's served under dev server (`retype watch`) or something else like `Gitlab Pages`. It happens always.

What difference between direct opening and following the link, Idk, since I don't know your code base, but I might suggest you to just add something like `Prism.highlightElement(element)` after you replace tab body

**Comment by geoffreymcgill** [comment]
This issue has been fixed and will be included in the Retype `v1.9.0` release. 

The issue turned out to be more complex than first thought. Isn't that always the case? Anyways, we've come up with a clean solution and it's working well. Both Prism syntax highlighting and [Mermaid](https://retype.com/components/mermaid/) diagrams were affected, but all will be well as of `v1.9.0`.

There are no changes required in your project. Only upgrading to `v1.9.0` once it's released.

@AdamBrianBright Thanks again for reporting the issue.
